tests/test_fdjump.py

```diff
@@ -17,6 +17,8 @@ def model_and_toas():
         "FD1JUMP -sys GM_GWB_1460_100_b1 0.02 1\n"
         "FD2JUMP -sys GM_GWB_500_100_b1 0.001 1\n"
         "FD2JUMP -sys GM_GWB_1460_100_b1 0.002 1\n"
+        "FDJUMPDM -sys GM_GWB_500_100_b1 0.00002 1\n"
+        "FDJUMPDM -sys GM_GWB_1460_100_b1 0.00001 1\n"
```

Comment:
Do you think it would be worth including lines like these in the docstring or in `explanation.rst`, just to give a concrete example?